readme.md

```diff
@@ -383,21 +436,22 @@ juan.full_name
 
 `attr_accessor` creates getters and setters.
 
-## Break  (10:50 - 11:00, 10 min)
+-------
+
+## Break (10 minutes / 1:00)
 
-___
+-------
 
-## You do: Monkies! (11:00 - 11:20, 20 min)
+## You Do: Monkies! (20 minutes / 1:20)
 
 For the next exercise, clone down the repo linked below:
 https://github.com/ga-wdi-exercises/oop_monkey
 
+-------
 
-## Class-level stuff
-
```

Comment:
by this point, im feelin overwhelmed about all the terms. can we get some sort of diagram about local vs instance vs class vars an methods